src/sync/partial_sync.cpp

```diff
@@ -643,12 +642,16 @@ Subscription subscribe(Results const& results, util::Optional<std::string> user_
     auto query = results.get_query().get_description(); // Throws if the query cannot be serialized.
     query += " " + results.get_descriptor_ordering().get_description(results.get_query().get_table());
 
-    std::string name = user_provided_name ? std::move(*user_provided_name)
-                                          : default_name_for_query(query, results.get_object_type());
+    if (options.inclusions.is_valid()) {
+        query += " " + options.inclusions.get_description(results.get_query().get_table());
```

Comment:
Minor, but the unconditional padding of empty whitespace is a bit annoying (especially when writing tests), now, if both descriptor ordering and inclusions are empty you end up with two empty " " at the end of the query string.

Reply:
@cmelchior I've updated things so that there is no unnecessary whitespace at the end of subscriptions without sort/distinct/limit. You'll have to update any binding tests that check subscription syntax, but I figure better to take care of this now than have it as a continual annoyance.